crates/re_renderer/src/renderer/outlines.rs

```diff
@@ -87,6 +87,22 @@ impl OutlineMaskPreference {
     pub fn is_none(self) -> bool {
         self.0.is_none()
     }
+
+    #[inline]
+    pub fn combine(self, other: Self) -> Self {
```

Comment:
A comment explaining the intended "combine" logic would be helpful.

Seems like self.a and self.b always take precedent unless they are None or 0 in which case we fall back to other?